backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/workplace/WorkplaceRepository.java

```diff
@@ -111,10 +111,25 @@ public Workplace create(@NonNull final WorkplaceCreateRequest request)
 		record.setM_Warehouse_ID(request.getWarehouseId().getRepoId());
 		record.setPickFrom_Locator_ID(LocatorId.toRepoId(request.getPickFromLocatorId()));
 		record.setM_PickingSlot_ID(PickingSlotId.toRepoId(request.getPickingSlotId()));
+		record.setMaxPickingJobs(request.getMaxPickingJobs());
 
-		final Workplace workplace = getAllActive().stream().max(Comparator.comparing(v -> v.getSeqNo().toInt())).orElse(null);
-		final SeqNo seqNo = workplace != null ? workplace.getSeqNo().next() : SeqNo.ofInt(0);
+		final SeqNo seqNo;
+		if(request.getSeqNo() != null)
+		{
+			seqNo = request.getSeqNo();
+		}
+		else
+		{
+			final Workplace workplace = getAllActive().stream().max(Comparator.comparing(v -> v.getSeqNo().toInt())).orElse(null);
```

Comment:
pls introduce WorkplacesMap.getNextSeqNo() method.

note: usually the first seqNO is 10 and not zero.